src/Raven.Server/Config/Categories/CoreConfiguration.cs

```diff
@@ -46,6 +46,12 @@ public class CoreConfiguration : ConfigurationCategory
         [ConfigurationEntry("ThrowIfAnyIndexCannotBeOpened", ConfigurationEntryScope.ServerWideOrPerDatabase)]
         public bool ThrowIfAnyIndexCannotBeOpened { get; set; }
 
+        [Description("Indicates request latency threshold before the server would issue an alert")]
+        [ConfigurationEntry("TooLongRequestThreshold", ConfigurationEntryScope.ServerWideOnly)]
+        [DefaultValue(2)]
```

Comment:
Let's set the default to 5, 2 seems too low.

Reply:
Should be moved to PerformanceHintsConfiguration? Key should be 'PerformanceHints.TooLongRequestThresholdInSec' for consistency.

Reply:
Yes, good idea